lib/helper_patches.rb

```diff
@@ -17,4 +17,6 @@ def request_url_with_query
       uri.to_s
     end
   end
+
+  DatasetteHelper.datasette_url = 'https://data.mysociety.org/datasette/'
```

Comment:
I'm okay with this but would why like this instead of a `config/general.yml` configuration variable which we would normally use?